src/status_im2/navigation/screens.cljs

```diff
@@ -259,7 +260,13 @@
 
     {:name      :wallet-select-address
      :options   {:modalPresentationStyle :overCurrentContext}
-     :component wallet-select-address/view}]
+     :component wallet-select-address/view}
+
+    {:name      :scan-address
+     :options   (merge
```

Comment:
`assoc` is a better choice here.

Quick question: Can we use `transparent-screen-options` instead of `dark-screen`?